importexport/musicxml/exportxml.cpp

```diff
@@ -2400,6 +2400,13 @@ static Breath* hasBreathMark(Chord* ch)
       return s ? toBreath(s->element(ch->track())) : 0;
       }
 
+static Breath* hasBreathMark(Rest* rest)
```

Comment:
I guess it makes sense to move this method into Rest class

Reply:
Since we also use this for `Chord`s (`static Breath* hasBreathMark(Chord* ch)`, line 2396), would it make sense to move this method into the `ChordRest` class?

Reply:
Makes sense for me